src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedbackContent/WidgetFeedbackContent.tsx

```diff
@@ -1,8 +1,6 @@
-// @flow
-
 import React, { type ComponentType } from 'react';
 import { spacersNum, colors } from '@dhis2/ui';
-import { withStyles } from '@material-ui/core';
```

Comment:
use the pattern` import { withStyles, type WithStyles } from '@material-ui/core';` from src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentQuickActions/QuickActionButton/QuickActionButton.tsx